README.md

```diff
@@ -32,57 +32,60 @@ See [this setup guide](https://github.com/stitchfix/eng-wiki/blob/master/technic
 For Rails projects, we provide a Railtie that automatically configures and loads Log Weasel.
 
 To see Log Weasel transaction IDs in your Rails logs either use the Logger provided or
-customize the formatting of your logger to include <code>LogWeasel::Transaction.id</code>.
+customize the formatting of your logger to include `LogWeasel::Transaction.id`.
 
-<pre>
+```rb
 YourApp::Application.configure do
   config.log_weasel.key = 'YOUR_APP'    # Optional. Defaults to Rails application name.
 end
-</pre>
+```
 
 ## Javascript
 
 If you have added the optional NPM package, this is how you would use it:
 
-<pre>
+```js
 import LogWeasel from "@stitch-fix/log-weasel";
+// Get the canonical fixops app name from your app's package.json
+import { name } from '../../package.json';
```

Comment:
❤️  forgot about using the name defined in `package.json` - nice.

Reply:
Wait, is there always an app name defined in `package.json`?

Reply:
Looks like it's required. If someone is missing one, that's definitely something they should fix.

https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields

Reply:
@brettfishman And when using the app template, I believe the name in package.json will match the app's name in fixops!